types/timed-cache/index.d.ts

```diff
@@ -1,23 +1,51 @@
 export interface CacheOptions {
-    defaultTtl: number;
+    /**
+     * Specify what default value you would like the TTL to have when creating the storage
+     * @default 60 * 1000
+     */
+    defaultTtl?: number;
 }
 
-export interface PutOptions {
-    ttl: number;
+export interface PutOptions<T> {
+    /**
+     * You can customize the time-to-live value of a key/value pair at insertion time
+     */
+    ttl?: number;
```

Comment:
I suspect the options here also need `| undefined` too like your other PRs.

Reply:
Fixed.